asciidoc_reader/asciidoc_reader.py

```diff
@@ -74,7 +74,7 @@ def _read_metadata(self, source_path):
         """Parses the AsciiDoc file at the given `source_path` and returns found
         metadata."""
         metadata = {}
-        with open(source_path) as fi:
+        with open(source_path, encoding="latin-1") as fi:
```

Comment:
Wait, `latin-1`? According to the AsciiDoctor FAQ, AsciiDoctor treats input files as UTF-8 by default and as UTF-16 if there's a BOM. Shouldn't we mimic the same behavior here?

Reply:
I don't like that either. The root of this problem is in the implementation of the `fix_unicode` function for `python3`.
Another way for me is to remove `val = str.encode(val, "latin-1"). decode("utf-8")` from `fix_unicode` function.

Reply:
I agree. Encoding should be `utf-8` and after that as far as I can see, `fix_unicode` can be removed. It shouldn't be necessary.

Reply:
Okay, done in second commit.